crates/ra_ssr/src/errors.rs

```diff
@@ -0,0 +1,29 @@
+//! Code relating to errors produced by SSR.
+
+/// Constructs an SsrError taking arguments like the format macro.
+#[macro_export]
+macro_rules! error {
```

Comment:
I think this doesn't need to be `#[macro_export]`, `#[macro_use]` on the module should be enough. We also use macro re-export idiom:

```
macro_rules! _my_macro {}
pub(crate) use _my_macro as my_macro; // can now use `my_macro` with usual name resolution rules. 
```

Reply:
Done